int16 dx
int16 dy
int16 dpsi
int16 dz
